### PR TITLE
Add ancient staff to prayer items list

### LINKED ISSUE #7508: Add ancient staff prayer bonus to prayer plugin
Please check if your issue is not a duplicate by [searching existing issues](https://github.com/runelite/runelite/search?type=Issues)

**Describe the bug**
The tooltip does not correctly display prayer bonus when wearing the Ancient Staff (-1 prayer)

**To Reproduce**
1. Look at prayer bonus on the Activate quick-prayers tooltip
2. Wear Ancient Staff
3. Prayer bonus does not decrease by 1

**Expected behavior**
Tooltip should display prayer minus 1 when wearing Ancient Staff

**Screenshots**
https://i.imgur.com/kE5qdQL.png

**Environment (please complete the following information):**
 - OS: Windows 10 Education
 - RuneLite version: 1.5.8
 - Launcher version: 1.5.2

**Additional context**
none

### PR DESCRIPTION
**Pull request by lund8619**
Closes #7508

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerItems.java [hunk 1]
```diff
@@ -388,6 +388,7 @@
 	ZAMORAK_GODSWORD(ItemID.ZAMORAK_GODSWORD, 8),
 	ZAMORAK_GODSWORD_OR(ItemID.ZAMORAK_GODSWORD_OR, 8),
 	IVANDIS_FLAIL(ItemID.IVANDIS_FLAIL, 5),
+    ANCIENT_STAFF(ItemID.ANCIENT_STAFF, -1),
```
Comment (Spedwards) [inline runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerItems.java]: Correct indentation to a tab and move to the top of the list, as they are sorted by prayer bonus.

Comment (lund8619) [reply runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerItems.java]: Okay. I need to stop long enough to setup my defaults properly. Then I'll go reorder the list.

Comment (lund8619) [reply runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerItems.java]: Alright, the indentation issues should be done with now.

**Comment by lund8619** [comment]
For issue #7508

**Pull request merged**